js/models/messages.js

```diff
@@ -127,6 +127,7 @@
       }
 
       this.selected = false;
+      window.contextMenuShown = false;
```

Comment:
Surely there is a way to pass the state from the message model to the message view directly without storing it in the global scope? Using props or something?

Reply:
Yes, there is but it is *really* messy (because it would have to cross the backbone/react boundary) and will require updating every message...

Reply:
Fair enough, I know the UI side is a bit wacky.
In a perfect world, it seems like the context menu should be managed at the conversation level, not the message level, but, alas, this is inherited code.